assets/helpers/visitToken/actions.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+// ----- Types ----- //
+
+export type Action = { type: 'SET_VISIT_TOKEN', name: string };
+
+
+// ----- Actions Creators ----- //
+
+export function setToken(name: string): Action {
+  return { type: 'SET_VISIT_TOKEN', name };
+}
+
+export default setToken;
```

Comment:
We prefer [named exports](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#Description) to default exports for everything except components and reducer files. This is because these types of files conceptually should only ever have one export, whereas other files, such as those that contain actions, can have multiple exports. So in this case:

```jsx
export { setToken };
```

Reply:
What's the rationale for preferring exports at the bottom of the file? I find it easier to see right next to the function body whether it's exported (it's akin to marking a method `public` in other languages), rather than having to the jump to the bottom of the file. It can sometimes make the cmd+clicking in IntelliJ a bit more tortuous

Reply:
this file has been deleted in the next PR, but interesting debate, thanks for the explanation